### DISCUSSION TITLE
Behavior of Social provider logins and setting a password

### DISCUSSION THREAD

**Discussion by leafblader1** [supabase/supabase#5401] [Questions] [answered]
What is the expected behavior of a social provider logging in? I've noticed that users can log in via Discord, and then update their password and be able to log in again via normal username/password. Is this something to be concerned of, or should it be described as a way to both log in via email + discord, if you set the password?

**Comment by GaryAustin1** [comment, marked as answer]
Unless you use phone auth (not really sure of details on it), all the 3rd party auth providers, and the email/password interface, use an  email address as unique identifier.  So a user is the same no matter the method of login if email is same.
